node-manager/src/node.rs

```diff
@@ -476,10 +474,7 @@ impl Node {
         let amount_msat = amount_sat.map(|s| s * 1_000);
         let invoice = match route_hints.len() {
             0 => {
-                let now = SystemTime::now()
-                    .duration_since(SystemTime::UNIX_EPOCH)
-                    .unwrap()
-                    .as_secs();
+                let now = crate::utils::now();
```

Comment:
```suggestion
                let now = crate::utils::now().as_secs();
```

Should that have `as_secs()` like the rest of them do?

Reply:
We don't need to here, before we were getting the seconds and then turning back into a duration. This just removes the extra conversion